ethcore/private-tx/src/private_transactions.rs

```diff
@@ -223,7 +223,7 @@ impl SigningStore {
 		&mut self,
 		private_hash: H256,
 		transaction: SignedTransaction,
-		validators: Vec<Address>,
+		validators: &Vec<Address>,
```

Comment:
Unfortunately these validators array is required in two different places, so I cannot just move it to the one of them.

Reply:
If necessary the `Vec` can be cloned on the call site, no? My point was that `add_transaction` needs `validators` by value.

Reply:
Yes, it could be cloned on the `call-site` but maybe @grbIzl is concerned by the massive call overhead of passing 3 words instead 1 :smile: 

Also, I noticed that transaction is cloned which is not needed because it moved into function but it is probably out-of-scope for this PR